corehq/apps/reports/models.py

```diff
@@ -208,8 +208,8 @@ class ReportsSidebarOrdering(models.Model):
 
 class TableauServer(models.Model):
     SERVER_TYPES = (
-        ('server', 'Tableau Server'),
-        ('online', 'Tableau Online'),
+        ('server', _('Tableau Server')),
```

Comment:
These need to use `ugettext_lazy` because this code is run when this file is loaded (as opposed to it being inside a function call) and at that point there isn't a user so it's unknown what language to translate into. Check out the description of `ugettext_lazy` here: https://commcare-hq.readthedocs.io/translations.html